### PR TITLE
Adding Akaunting

### PR DESCRIPTION
**Pull request by pubmania**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x ] I have tested the template using the method described in README.md thoroughly
- [ x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [ x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [ x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [ x] Icon is added as a png file to the logos directory.

### CODE REVIEW

**Comment by pubmania** [comment]
Hi, Please can you help on what the issue is here and why the check is failing. I don't understand the error with formatter. I have checked, everything is as it should be, and it does work on my own instance.

File: public/v4/apps/akaunting.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+services:
+    # Akaunting
+    $$cap_appname:
+        depends_on:
+           - $$cap_appname-db
+        image: akaunting/akaunting:$$cap_app_version
+        build:
+          context: .
+        ports:
+            - 8080:80
```
Comment (githubsaturn) [inline public/v4/apps/akaunting.yml:11]: You don't need this. CapRover automatically reverse proxy to your app.

File: public/v4/apps/akaunting.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            COMPANY_NAME: $$cap_company_name
+            COMPANY_EMAIL: $$cap_company_email
+            ADMIN_EMAIL: $$cap_admin_email
+            ADMIN_PASSWORD: $$cap_admin_password
+        volumes:
+            - $$cap_appname-data:/akaunting-data
+            - $$cap_appname-modules:/akaunting-modules
+    # Database
+    $$cap_appname-db:
+        image: mariadb
```
Comment (githubsaturn) [inline public/v4/apps/akaunting.yml:32]: Specify the db version here.

**Review by githubsaturn — COMMENTED** [review]
Looks good.

Regarding the formatting, you can simply run
```
npm i
npm run formatter-write
```

**Review by pubmania — COMMENTED** [review]
made changes as per the suggestion

**Comment by pubmania** [comment]
> Looks good.
> 
> Regarding the formatting, you can simply run
> 
> ```
> npm i
> npm run formatter-write
> ```

oh... I dont know how to do that ... I have done this all on the github website so where should I run that command?

**Comment by githubsaturn** [comment]
Thanks! I'll fix the formatting after merge.

**Pull request merged**